Application crashes if no balance is found for Villabanken

The application will crash if no balance is found for Villabanken.

```java
java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.IllegalStateException: No successful match so far
       at java.util.regex.Matcher.ensureMatch(Matcher.java:596)
       at java.util.regex.Matcher.group(Matcher.java:357)
       at com.liato.bankdroid.banking.banks.Villabanken.update(Villabanken.java:177)
       at com.liato.bankdroid.DataRetrieverTask.doInBackground(DataRetrieverTask.java:89)
       at com.liato.bankdroid.DataRetrieverTask.doInBackground(DataRetrieverTask.java:40)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

and 

```java
java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.IllegalStateException: No successful match so far
       at java.util.regex.Matcher.ensureMatch(Matcher.java:471)
       at java.util.regex.Matcher.group(Matcher.java:578)
       at com.liato.bankdroid.banking.banks.Villabanken.update(Villabanken.java:169)
       at com.liato.bankdroid.BankEditActivity$DataRetrieverTask.doInBackground(BankEditActivity.java:235)
       at com.liato.bankdroid.BankEditActivity$DataRetrieverTask.doInBackground(BankEditActivity.java:216)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
```

--- a/bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Villabanken.java
+++ b/bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Villabanken.java
@@ -165,35 +165,34 @@ public class Villabanken extends Bank {
 		Matcher matcher;
 
         matcher = reDisposableAmount.matcher(accountResponse);
-        matcher.find();
-        Account account = new Account("Disponibelt belopp", Helpers.parseBalance(matcher.group(2)), "0");
-        account.setType(Account.CCARD);
-        account.setCurrency(currency);
-        accounts.add(account);
-        balance = balance.add(account.getBalance());
-
+        if(matcher.find()) {
+            Account account = new Account("Disponibelt belopp", Helpers.parseBalance(matcher.group(2)), "0");
+            account.setType(Account.CCARD);
+            account.setCurrency(currency);
+            accounts.add(account);
+            balance = balance.add(account.getBalance());
+        }
         matcher = reBalance.matcher(accountResponse);
-        matcher.find();
-        account = new Account("Saldo", Helpers.parseBalance(matcher.group(2)), "1");
-        account.setType(Account.OTHER);
-        account.setAliasfor("Saldo alias");
-        account.setCurrency(currency);
-        accounts.add(account);
-
+        if(matcher.find()) {
+            Account account = new Account("Saldo", Helpers.parseBalance(matcher.group(2)), "1");
+            account.setType(Account.OTHER);
+            account.setAliasfor("Saldo alias");
+            account.setCurrency(currency);
+            accounts.add(account);
+        }
         matcher = reCreditLimit.matcher(accountResponse);
-        matcher.find();
-        account = new Account("Köpgräns", Helpers.parseBalance(matcher.group(2)), "2");
-        account.setType(Account.OTHER);
-        account.setAliasfor("Köpgräns alias");
-        account.setCurrency(currency);
-        accounts.add(account);
-
+        if(matcher.find()) {
+            Account account = new Account("Köpgräns", Helpers.parseBalance(matcher.group(2)), "2");
+            account.setType(Account.OTHER);
+            account.setAliasfor("Köpgräns alias");
+            account.setCurrency(currency);
+            accounts.add(account);
+        }
         if (accounts.isEmpty()) {
             throw new BankException(res.getText(R.string.no_accounts_found).toString());
         }
 
         super.updateComplete();
-
 	}
 
     @Override

